src/python/_io.cpp

```diff
@@ -93,17 +92,3 @@ make_fromroot_nestedvector(py::module& m, const std::string& name) {
      py::arg("initial") = 1024,
      py::arg("resize") = 1.5);
 }
```

Comment:
Logically, the methods could go to _ext.cpp, especially given that you add the src/awkward1/_io.py placeholder. Another option I see is to remove underscores to have io.cpp.

Reply:
Since it's coming from _ext now, I suppose there's no reason to move objects from one hidden module to another. (Users should not be directly calling functions like `fromroot_nestedvector`.)

Since "_io" is no longer a Python module name, I suppose there's no point in keeping the underscore. Renamed.